examples/pxScene2d/src/rcvrcore/http_wrap.js

```diff
@@ -18,470 +18,12 @@ limitations under the License.
 
 'use strict';
 
-var isV8=(typeof _isV8 !== "undefined");
-
+var isHttp2Present = false;
```

Comment:
Please revert all changes to http_wrap (s). You are readding some old file which is out of date and doesn't have many other commits. http_wrap.js should work with node 6 and node 8, what is the problem you have with it?